app/src/main/java/org/oppia/android/app/settings/profile/ProfileRenameActivity.kt

```diff
@@ -7,6 +7,7 @@ import org.oppia.android.app.activity.ActivityComponentImpl
 import org.oppia.android.app.activity.InjectableAppCompatActivity
 import javax.inject.Inject
 
+/** Argument key for the Profile Id for which the name is going to be changed in [ProfileRenameActivity]. */
```

Comment:
Nit
`/** Argument key for the profile id for which the name is going to be changed in [ProfileRenameActivity]. */`